signature-aggregator/aggregator/aggregator.go

```diff
@@ -101,6 +105,7 @@ func (s *SignatureAggregator) AggregateSignaturesAppRequest(
 			zap.String("warpMessageID", unsignedMessage.ID().String()),
 			zap.Error(err),
 		)
+		s.metrics.ValidatorFailures.Inc()
```

Comment:
We probably want to distinguish between issues for more specific alerting. I.e. not connecting to validators is a broader issue than not being able to connect to sufficient stake and is very different from an individual validator not returning a valid signature response

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/398/commits/eb2e526cdda8fb63744c7e6bc460648b5b9b1889